src/test/java/selenium/Locators.java

```diff
@@ -0,0 +1,21 @@
+package selenium;
+
+import org.openqa.selenium.By;
+
+public class Locators {
+    public static By xpath = By.xpath("//input[@id='user-name']");
+    public static By id = By.id("user-name");
+    public static By cssSelector = By.cssSelector("#user-name");
+    public static By userName = By.name("user-name");
```

Comment:
no need to create 4 types of locators on each web element

Reply:
Done